apps/bluetooth-music/app.js

```diff
@@ -28,27 +29,30 @@ function resetLastUrl () {
 }
 
 function onAudioFocusGained () {
-  logger.debug('onAudioFocusGained')
+  logger.info('++++ onAudioFocusGained')
```

Comment:
Hey @morrowind, may I ask you to use more descriptive words instead?

Reply:
It's just for easy watching while debugging.